vignettes/advs.Rmd

```diff
@@ -342,8 +342,8 @@ advs <- advs %>%
 ```{r, eval=TRUE, echo=FALSE}
 dataset_vignette(
   arrange(advs, USUBJID, VISITNUM, VSTPTNUM, PARAMCD),
-  display_vars = exprs(USUBJID, PARAMCD, PCHG, CRIT1, CRIT1FL, CRIT2, CRIT2FL),
-  filter = PARAMCD %in% c("WEIGHT")
+  display_vars = exprs(USUBJID, PARAMCD, PCHG, CRIT1, CRIT1FL, CRIT2, CRIT2FL, VISIT, VISITNUM),
+  filter = PARAMCD %in% c("WEIGHT") & USUBJID == "01-701-1034"
 )
```

Comment:
@manciniedoardo  Added the filter for subject 1034 to display rows with CRTI1FL = "Y". If this is something that we want to show.

Reply:
I think that is a good idea. Perhaps also include someone without a critflag, just to show that there is a difference.